api/provider/aws/service_create.go

```diff
@@ -12,6 +12,20 @@ func (s *ServiceProvider) Create(req models.CreateServiceRequest) (string, error
 	fqEnvironmentID := addLayer0Prefix(s.Config.Instance(), req.EnvironmentID)
 	cluster := fqEnvironmentID
 
+	var securityGroupIDs []*string
+	environmentSecurityGroupName := getEnvironmentSGName(fqEnvironmentID)
+	environmentSecurityGroup, err := readSG(s.AWS.EC2, environmentSecurityGroupName)
```

Comment:
[nit] isn't this only required if the `launchType` is fargate? If so, it would be a little more efficient to do this only in that case.